lib/s3middleware/userMetadata.ts

```diff
@@ -14,7 +14,7 @@ export function getMetaHeaders(headers: http.IncomingHttpHeaders) {
         if (!v) return length;
         return length + k.length + v.toString().length;
     }, 0);
-    if (totalLength < constants.maximumMetaHeadersSize) {
+    if (totalLength <= constants.maximumMetaHeadersSize) {
```

Comment:
Ideally there's a unit test to be done here, so this doesn't happen in the future